### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuliiaSaliuta**

### CODE REVIEW

**Review by skoval7 — APPROVED** [review]
everything ok , but you have some problem with flake8

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]
All checks have failed

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```
Comment (LiudmylaKulinchenko) [inline app/main.py:3]: You can split each parameter into one line to increase readability

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done! 😋☀🌼

**Pull request closed without merging**